src/app/data-model/Nullable.h

```diff
@@ -37,31 +37,40 @@ inline constexpr auto NullNullable = NullOptional;
  * things.
  */
 template <typename T>
-struct Nullable : protected Optional<T>
+struct Nullable : protected std::optional<T>
 {
+
     //
     // The following 'using' statement is needed to make visible
     // all constructors of the base class within this derived class.
     //
-    using Optional<T>::Optional;
+    using std::optional<T>::optional;
+    using std::optional<T>::operator*;
```

Comment:
Do we want to pull in `operator*`?  I guess we can...  I worry about people misusing the unchecked thing.

Reply:
at least for using std::optional directly, clang-tidy was super aggressive on complaining about unchecked accesses. Beyond that however, this seems to have flash benefits.

Reply:
I would lean towards keeping it: std::optional provides this, so it seems others assumed it reasonable enough to have, so we may follow suite.